### PR TITLE
Fix stream in Plexon 1 interface

### PR DESCRIPTION
**Pull request by h-mayorquin**
Neo released, this is ready.

Should fix https://github.com/catalystneuro/neuroconv/issues/1000.

Related neo PR:
https://github.com/NeuralEnsemble/python-neo/pull/1524

### CODE REVIEW

**Comment by h-mayorquin** [comment]
Ah, the testing set that we have is too old. It only has this stream `V1` that is not documented. I send a couple of emails to inquire what is the nature of it but maybe we will need a new testing file.

**Comment by h-mayorquin** [comment]
This is ready, thanks for help me debug that silly mistake with the filename @CodyCBakerPhD .

File: src/neuroconv/datainterfaces/ecephys/plexon/plexondatainterface.py [hunk 1]
```diff
@@ -36,8 +43,18 @@ def __init__(self, file_path: FilePath, verbose: bool = True, es_key: str = "Ele
         verbose : bool, default: True
             Allows verbosity.
         es_key : str, default: "ElectricalSeries"
+        stream_name: str, optional
```
Comment (bendichter) [inline src/neuroconv/datainterfaces/ecephys/plexon/plexondatainterface.py:46]:
```suggestion
        stream_name: str, default="WB-Wideband"
```

File: src/neuroconv/datainterfaces/ecephys/plexon/plexondatainterface.py [hunk 2]
```diff
@@ -36,8 +43,18 @@ def __init__(self, file_path: FilePath, verbose: bool = True, es_key: str = "Ele
...
+            Only pass a stream if you modified the channel prefixes in the Plexon file and you know the prefix of
+            the wideband data.
         """
-        super().__init__(file_path=file_path, verbose=verbose, es_key=es_key)
+
+        if stream_name is None:
+            stream_name = "WB-Wideband"
```
Comment (bendichter) [inline src/neuroconv/datainterfaces/ecephys/plexon/plexondatainterface.py:52]:
```suggestion
```

File: src/neuroconv/datainterfaces/ecephys/plexon/plexondatainterface.py (hunk 1 above)
Comment (h-mayorquin) [reply src/neuroconv/datainterfaces/ecephys/plexon/plexondatainterface.py:45]: Changed.

**Review by bendichter — APPROVED** [review]

**Pull request merged**